LOGS-README.md

```diff
@@ -0,0 +1,118 @@
+Note: Be sure to replace <your-GCP-project-ID> with your own GCP project ID.
+```shell script
+gcloud iam service-accounts add-iam-policy-binding --role roles/iam.workloadIdentityUser --member "serviceAccount:<your-GCP-project-ID>.svc.id.goog[dynatrace/dynatrace-gcp-function-sa]" dynatrace-gcp-function-sa@<your-GCP-project-ID>.iam.gserviceaccount.com
+```
+
+4. Create `dynatrace-gcp-fuunction` IAM role(s).
+ Note: Be sure to replace <your-GCP-project-ID> with your own GCP project ID.
+
+for log ingest:
+```shell script
+wget https://raw.githubusercontent.com/dynatrace-oss/dynatrace-gcp-function/custom-gcp-roles/gcp_iam_roles/dynatrace-gcp-function-logs-role.yaml
+gcloud iam roles create dynatrace_function.logs --project=<your-GCP-project-ID> --file=--file=dynatrace-gcp-function-logs-role.yaml
+````
+
+for metrics ingest:
+```shell script
+wget https://raw.githubusercontent.com/dynatrace-oss/dynatrace-gcp-function/custom-gcp-roles/gcp_iam_roles/dynatrace-gcp-function-metrics-role.yaml
+gcloud iam roles create dynatrace_function.metrics --project=<your-GCP-project-ID> --file=dynatrace-gcp-function-metrics-role.yaml
+```
+
+5. Grant the required IAM policies to the service account.
+Note: Be sure to replace <your-GCP-project-ID> with your own GCP project ID.
+
+For logs ingest only:
+```shell script
+gcloud projects add-iam-policy-binding <your-GCP-project-ID> --member="serviceAccount:dynatrace-gcp-function-sa@<your-GCP-project-ID>.iam.gserviceaccount.com" --role=roles/dynatrace_function.logs
+```
+
+For logs and metrics ingest:
+```shell script
+gcloud projects add-iam-policy-binding <your-GCP-project-ID> --member="serviceAccount:dynatrace-gcp-function-sa@<your-GCP-project-ID>.iam.gserviceaccount.com" --role=roles/dynatrace_function.logs
+gcloud projects add-iam-policy-binding <your-GCP-project-ID> --member="serviceAccount:dynatrace-gcp-function-sa@<your-GCP-project-ID>.iam.gserviceaccount.com" --role=roles/dynatrace_function.metrics
+```
+
+6. Enable the APIs required for monitoring.
+```shell script
+gcloud services enable cloudapis.googleapis.com monitoring.googleapis.com cloudresourcemanager.googleapis.com
+```
+
+7. Install dynatrace-gcp-funtion with helm chart.
+
+Download helm chart package
+```shell script
+wget https://github.com/dynatrace-oss/dynatrace-gcp-function/releases/latest/download/dynatrace-gcp-function.tgz
+```
+
+Update values file with your configuration (deployment type, )
+see comments in [values.yaml](https://raw.githubusercontent.com/dynatrace-oss/dynatrace-gcp-function/feature/log-forwarder/k8s/helm-chart/dynatrace-gcp-function/values.yaml) for more details
+```shell script
+tar -zxvf dynatrace-gcp-function.tgz dynatrace-gcp-function/values.yaml
+vi dynatrace-gcp-function/values.yaml
+```
+
+Install helm chart
+```shell script
+helm install -f dynatrace-gcp-function/values.yaml dynatrace-gcp-function.tgz --generate-name
```

Comment:
Installation command is correct but if client unpacks helm chart to update values then command could be simpler:
helm install ./dynatrace-gcp-function --generate-name
or we could describe second option with --set: helm install <parameterName>=<value> dynatrace-gcp-function.tgz --generate-name 
but I agree that the second option is less readable when there are too many parameters.